docs/requirements/state-of-the-art.md

```diff
@@ -2,7 +2,19 @@
 
 ## Вступ
 
-*[Вступ повинен містити короткий огляд всього документу.]*
+**BranchOut** — інноваційна система управління проєктами, що спрощує процеси планування та виконання завдань як для технічних, так і для нетехнічних користувачів. Вона призначена для оптимізації роботи над проєктами будь-якої складності, пропонуючи інструменти для повного контролю над процесом розробки.
+
+**Мета BranchOut** — вирішення критичних недоліків інших систем управління проєктами, таких як GitHub, Jira та Asana. Сучасні рішення мають переважну орієнтацію на розробників, тож вони страждають від складності використання для нових користувачів, обмеженої підтримки кастомізації та слабкої адаптації до короткострокових завдань. BranchOut націлена на надання гнучкості та інтеграції, при мінімальних затратах на навчання користувачів.
+
+**Технічні характеристики розробки**:
+- Інтуїтивно зрозумілий інтерфейс
+- Система контролю версій для відстеження змін
+- Інтеграція SQL та REST API для зручної роботи з базами данних
+
+**Очікуваний технічно-економічний ефект**:
+- Підвищення продуктивності команд завдяки простоті та зручності використання
+- Розширення аудиторії завдяки підвищеній доступності
+- Зниження витрат на технічну підготовку
```

Comment:
move this introduction to `docs/intro/README.md`, here you should write a short overview of `state-of-art.md`